Sessionless Product Detail.md

```diff
@@ -0,0 +1,50 @@
+# Sessionless Product Details
+
+**Introduction & Base Definitions**
+
+
+Authentication protocols (auth protocols) are gatekeepers that verify the identity of a person to give them access to specific programs or data. They include a set of rules or procedures that govern how data is transmitted across electronic devices. The most common uses of auth protocols are passwords, but others include one-time-passwords, biometrics, and magic links. 
+
+- An Application Programming Interface (API) is  a software intermediary that allows two  applications to share data with each other
+- An API Key is  a unique code that identifies an application or user
+- An API Token is a piece of code that contains information about a user and is used to secure access to an API
+- Encryption is the process of turning data into code to prevent unauthorized access
```

Comment:
```suggestion
- Encryption is the process of encoding data to prevent unauthorized access
```

Reply:
My only concern here is that some people may not understand what "encoding" means.

Reply:
that's fair. My issue is that "turning data into code" implies code to me. e.g. 

```
if(true) {
    doSomething();
}
```

maybe just "Encryption is the process of preventing data from unauthorized access"

Reply:
I agree, the `code` word might be confusing and I also see it that way.

How about:
```
Encryption is the process of converting information into a secret format to prevent unauthorized access
```